housenergy-sample

Enteli opened the pull request:
Added houseenergy python sample from https://github.com/BonsaiAI/cs-house-energy


PR track: https://music.youtube.com/watch?v=lZ8zu1aRYA8&feature=share

journeyman-msft commented:
@Enteli Recall we discussed House Energy would be a direct PR into microsoft bonsai api?

Enteli commented:
@journeyman-msft  I hadn't realized by copying the houseenergy folder it had copied the .git and created a submodule  corrected.

1. Deleted cs-house-energy
2. Created a new directfolder (no submodule) named house-energy
3. renamed `__main__.py` to `main.py`
4. updated DockerFile and DockerFile-windows
5. created an environment.yml file and tested

Enteli commented:
@nmann4 can I ask you to review this PR? Just added house-energy python sample, by following the README, you should be able to run the sample

nmann4 commented on Python/samples/house-energy/sim/house_simulator.py:
```diff
@@ -0,0 +1,142 @@
+        for i in range(288):
+            house.update_hvacON(random.randint(0, 1))
+            house.update_Tin()
+            #print(house.Tin)
```
Is this meant to be here? Just curious because it's commented out. It would probably be best to remove it as it looks like it's used for debugging.

Enteli replied:
Thank you @nmann4, uncommented and pushed. Merging

nmann4 reviewed (approved):
Everything looks good to me. I approve. Left a comment.
